play/PlayerRules/Basic Rules.txt

```diff
@@ -441,40 +472,53 @@ to subdue your opponent (either through choke-out, pin, or a joint lock). You
 may also strike an opponent during grappling, but if the strike fails your 
 opponent may attempt to gain control of the grapple. Skill in grappling is 
 different than skill in unarmed combat.
+Instead of dealing full damage, a character can try to kick/shove a human sized
+opponent to force movement by rolling for a normal unarmed attack with a -2
+miss penalty. If the attack succeeds, it yields 1/2 the damage it normally 
+would and pushes the target 1m for every 5 strength you have.
 
 Actions:
 During combat, your character is given 4 Actions to use for fighting, 
-movement, using items, etc. Here is a table of common tasks and their
+movement, using items, etc. Here is a list of common tasks and their
 relative cost:
 
 Reloading a Pump-Action Shotgun - 4 Actions
-Reloading a Bolt-Action weapon - 3 Actions
-Moving DEX meters - 2 Actions
-Hunkering Down - 2 Actions
-Firing a semi-auto gun - 1 Action
+Reloading a Bolt-Action Weapon - 3 Actions
+Reloading a Magazine Fed Weapon - 2 Actions
+Moving DEX # of Meters - 2 Actions
+Hunkering Down - 1 Action
+Firing a Semi-Auto Gun - 1 Action
+Suppressing an Area/Target - 1 Action
 Cast a Spell - 1 Action
-Aiming Down Sights - 1 Action
-turning on a flashlight - 1 Action
-Unholstering Pistol - 1 Action
-Performing a skill check (other than Awareness) - 1 Action
-Conversing with partners - Free Action
-Getting your bearings / non skill-based perception - Free Action
+Aiming Down Gun Sights - 1 Action
+Turning on a Flashlight - 1 Action
+Unholstering Pistol or Grabbing a Weapon/Item - 1 Action
+Performing a Skill Check (other than Awareness/Perception) - 1 Action
+Conversing With Partners - Free Action
+Getting Your Bearings / Non Skill-Based Perception - Free Action
 
 Movement:
 You can run DEX number of meters per 2 actions. A character with 1 DEX can 
 move 1m per 2 actions, or 2m per 4 actions (a full turn). You can sprint 
 instead of running, for a +2 movement bonus per two actions. However,
-firing your weapon after doing so will cause you to incur a +30% firing penalty.
-The penalty occurs during the actions you sprint, as well as 2 actions 
-afterwards. You can stack sprinting a number of Actions equal to your Fortitude; 
-however, for each Action taken sprinting, an Action of recovery (and resulting 
-+30% weapon penalty) must be taken after the sprinting ends.
+firing your weapon after doing so will cause you to incur a +1 miss chance
+penalty. The penalty occurs during the actions you sprint, as well as 2 actions 
+afterwards. You can stack sprinting a number of Actions equal to your Fortitude.
 You can jump horizontally Dex/2 meters, and add 1 meter if you have a 5 meter 

```

Comment:
Please reword 'stack sprinting actions' to 'sprint for a number of subsequent actions equal to' for the sake of clarity